### PR TITLE
Lj 266 card component

### PR DESCRIPTION
**Pull request by MarcoGoC**
# Description

[LJ-266] Card Component 

Added the new Card component (based on the topicbox) changes to the page to make it look like 
the new prototype page

## Test Instructions

1. Download the branch and run 

## Meets Definition of Done

- [ ] Meets design spec
- [ ] unit tests are included

[LJ-266]: https://lj-decd.atlassian.net/browse/LJ-266

### CODE REVIEW

**Comment by P0NDER0SA** [comment]
A couple things i noticed... when i ran it i see this warning for the card:
image

I'm wondering if we couldn't adjust the border as well (maybe remove it) to sharpen the look a bit -- and there are a couple of code recommendations (unused imports, etc) that the LGTM bot mentions above.  These are all very very minor though - so up to you if you want to push and fix these as a separate item.

File: components/atoms/Card.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import PropTypes from "prop-types";
+
+/**
+ *  Card Component
+ */
+
+export default function Card(props) {
+  return (
+    <div className="text-black text-left card rounded-tl-md rounded-b-md border cursor-pointer my-6 mx-4">
+      <div className="">
```
Comment (krischarbonneau) [inline components/atoms/Card.js:12]: can probably remove the className portion of this since it's empty

**Review by krischarbonneau — CHANGES_REQUESTED** [review]
I think this looks good for the most part, looking at the figma designs though there is far too much padding between the title and the service. I think we should make the service more of a subtitle like the designs.
Screenshot 2021-07-13 131834
Screenshot 2021-07-13 131817

**Review by P0NDER0SA — APPROVED** [review]
My stuff was addressed!  Looks good... just check out Kris' stuff now i guess

**Comment by MarcoGoC** [comment]
> I think this looks good for the most part, looking at the figma designs though there is far too much padding between the title and the service. I think we should make the service more of a subtitle like the designs.
> Screenshot 2021-07-13 131834
> Screenshot 2021-07-13 131817

Problem is that the cards look swirly (up and down) when the title is in two lines that pushes the links down and all the cards are  uneven. I'll try something else hopefully it wont look as bad

**Pull request merged**